### PR TITLE
Some fixes for the filters 

### PR DESCRIPTION
**Pull request by gwyneplaine**
- We now auto focus the first input in the dialog as per https://www.w3.org/TR/wai-aria-practices-1.1/#dialog_modal
- We now also update the filter on 'enter' keydown.

### CODE REVIEW

**Comment by gautamsi** [comment]
We were missing a filter `is_null: true ` for all nullable field.

Can you add it if not already.
@timleslie @JedWatson

**Comment by JedWatson** [comment]
@gautamsi that's on our list, we're in the middle of a big change to fields that I've just written up here: #5635

As part of that, we're going to be updating filters across all our field types

**Comment by gautamsi** [comment]
I see big changes, thanks for heads up

File: packages-next/admin-ui/src/pages/ListPage/FilterAdd.tsx [hunk 1]
```diff
@@ -125,22 +125,32 @@ function FilterAddPopoverContent({ onClose, listKey }: { onClose: () => void; li
...
+      onClose();
+    }
+  };
+
   return (
     <Stack
       padding="medium"
       as="form"
+      onKeyDown={(event: React.KeyboardEvent<HTMLFormElement>) => {
+        if (event.code === 'Enter') {
```
Comment (mitchellhamilton) [inline packages-next/admin-ui/src/pages/ListPage/FilterAdd.tsx:145]: Why does an enter not trigger the submit event?

**Review by emmatown — COMMENTED** [review]
Needs a changeset

**Review by timleslie — APPROVED** [review]
This all looks good 👍

File: packages-next/admin-ui/src/pages/ListPage/FilterAdd.tsx (hunk 1 above)
Comment (gwyneplaine) [reply packages-next/admin-ui/src/pages/ListPage/FilterAdd.tsx]: Ah... I thought it was react-select hijinx, but turns out its #justbuttonthings. 
Button didn't have an explicit type, so its hijacking form events in the onClick (which is also triggered by an enter key press) 

Fixed and pushed

**Review by emmatown — APPROVED** [review]

**Pull request merged**